docs/release-notes/Percona-Everest-1.1.0-(2024-08-12).md

```diff
@@ -0,0 +1,130 @@
+
+- [EVEREST-910](https://perconadev.atlassian.net/browse/EVEREST-910) - The schedule name and storage location for scheduled backups are now displayed on the UI.
+
+
+## Bugs fixed
+
+- [EVEREST-1233](https://perconadev.atlassian.net/browse/EVEREST-1233) - You will now see the correct error message when attempting to downgrade a major database version.
+
+- [EVEREST-740](https://perconadev.atlassian.net/browse/EVEREST-740) -  MySQL is now correctly selected as the default database engine when creating a database, even if it wasn't the first operator installed.
+
+- [EVEREST-1181](https://perconadev.atlassian.net/browse/EVEREST-1181) - The option to upgrade the major version of the database engine for MongoDB and PostgreSQL is now correctly disabled in the database edit section, reflecting the intended functionality.
+
+- [EVEREST-859](https://perconadev.atlassian.net/browse/EVEREST-859) - The issue causing an error during namespace deletion while uninstalling Percona Everest has been resolved.
+
+
+- [EVEREST-1074](https://perconadev.atlassian.net/browse/EVEREST-1074) - The backup page performance is now optimized for adding and editing backup.
+
+- [EVEREST-1141](https://perconadev.atlassian.net/browse/EVEREST-1141) - Backup files in the S3 bucket are now properly removed when the corresponding database is deleted.
+
+- [EVEREST-1144](https://perconadev.atlassian.net/browse/EVEREST-1144) - While editing the backup storage in a PostgreSQL database backup schedule, an error was encountered after three backup schedules were created. The issue has been resolved now.
+
+
+
+- [EVEREST-1050](https://perconadev.atlassian.net/browse/EVEREST-1050) - The restore page now correctly updates the restore information. 
+
+- [EVEREST-1244](https://perconadev.atlassian.net/browse/EVEREST-1244) - While attempting to restore a database, there was a discrepancy between the messages indicating the status of the restoration process and the actual actions being taken by Percona Everest. The issue has been resolved now.
+
+- [EVEREST-307](https://perconadev.atlassian.net/browse/EVEREST-307) - CLI errors now display more user-friendly messages without exceptions and stack traces.
+
+
+## Known limitations
+
+You can't use the same URL, bucket, and region in two different backup storages. Doing so may cause issues with restoring from the existing backups.
+
+Here’s what you need to know:
+
+**Scenario 1**
+
+If you have multiple storages with the same bucket, URL, and region, you won’t be able to edit them after the 1.1.0 update. You’ll need to delete the duplicates.
+
+
+To check whether your existing Everest installation has any backup storages using the same bucket, region, and endpoint URL, execute the following command:
+
+```sh
+curl "https://raw.githubusercontent.com/percona/everest-doc/main/tools/bin/check-duplicated-storages.sh" | bash
```

Comment:
To avoid the progress output we can use `-sS`.
```suggestion
curl -sS "https://raw.githubusercontent.com/percona/everest-doc/main/tools/bin/check-duplicated-storages.sh" | bash
```